tests/e2e/k8s/util/fabric.go

```diff
@@ -310,6 +311,16 @@ func (f *Fabric) DeployClusterlinks(peerCount uint8, cfg *PeerConfig) ([]*Cluste
 	return clusterlinks, nil
 }
 
+// GetPeerResources returns peer resources.
+func (f *Fabric) GetPeerResources(num int) *resources.Resources {
```

Comment:
I would define `GetPeerCluster(num int) *util.KindCluster` instead.

Reply:
But the resources field is not exposed.

Reply:
Add `Resources()` function to `KindCluster`.